parent/pom.xml

```diff
@@ -380,7 +380,7 @@
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-surefire-plugin</artifactId>
-                    <version>2.18.1</version>
+                    <version>3.0.0-M3</version>
```

Comment:
How is that relevant for this fix?

Reply:
Not relevant at all, I just thought I use the same PR to also update that.